### PR TITLE
RS-510: Rename entry with web console

### PR DESCRIPTION
**Pull request by AnthonyCvn**
Closes #

### Please check if the PR fulfills these requirements

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] CHANGELOG.md has been updated (for bug fixes / features / docs)

### What kind of change does this PR introduce?

feature

### What was changed?

As a user, I can rename entry with Web Console

### Related issues

No

### Does this PR introduce a breaking change?

Version 1.12: The rename method was introduced.

### CODE REVIEW

File: src/Views/BucketPanel/BucketDetail.test.tsx [hunk 1]
```diff
@@ -111,4 +112,46 @@ describe("BucketDetail", () => {
...
+    const emptyEntryRow = rows.at(1);
+
+    const renameIcon = emptyEntryRow.find('span[title="Rename entry"]');
+    renameIcon.simulate("click");
+
+    const renameModal = detail.find('div[data-testid="rename-modal"]');
+    const removeModal = detail.find('div[data-testid="delete-modal"]');
+
+    expect(renameModal.exists()).toBe(true);
+    expect(removeModal.exists()).toBe(false);
```
Comment (atimin) [inline src/Views/BucketPanel/BucketDetail.test.tsx:155]: let's check if the renameEntry method was called.

**Review by atimin — APPROVED** [review]
It's perfect.  Just add a check for the method call in the test.

File: src/Views/BucketPanel/BucketDetail.test.tsx (hunk 1 above)
Comment (AnthonyCvn) [reply src/Views/BucketPanel/BucketDetail.test.tsx:158]: I was struggling to simulate the input update with antd this morning but finally found a solution ! Now it should be good.

**Review by atimin — APPROVED** [review]
Perfect!

**Pull request merged**